package.json

```diff
@@ -2,25 +2,18 @@
 	"name": "package-json-validator",
 	"version": "0.10.0",
 	"description": "Tools to validate package.json files.",
-	"keywords": [
-		"package.json",
-		"validator",
-		"package.json validator",
-		"lint",
-		"package.json linter"
-	],
 	"homepage": "https://github.com/JoshuaKGoldberg/package-json-validator#readme",
 	"bugs": {
 		"url": "https://github.com/JoshuaKGoldberg/package-json-validator/issues"
 	},
 	"repository": {
 		"type": "git",
-		"url": "https://github.com/JoshuaKGoldberg/package-json-validator"
+		"url": "git+https://github.com/JoshuaKGoldberg/package-json-validator.git"
 	},
 	"license": "MIT",
 	"author": {
 		"name": "Nick Sullivan",
-		"email": "nick@sullivanflock.com"
+		"email": "npm@joshuakgoldberg.com"
```

Comment:
🐛 I think a `package.json`'s `author.email` should be prioritized over asking for the running user's `npm whoami` email. https://github.com/JoshuaKGoldberg/create-typescript-app/issues/2093